### PR TITLE
Feature/display puzzles

### PR DESCRIPTION
**Pull request by JRicardoT**
**What (if any) features are you implementing?**
- Fetched all puzzles to display in puzzles page.

**What (if anything) did you refactor?**
- Refactor the fetch request that was made initially.
- Refactor the data types being passed into `Puzzle` component.

**Were there any issues that arose?**
- None.

**Is there anything needs to be prioritized?**
- N/A.

**Next Steps?**
- Style the grid where all puzzles are being displayed.

### CODE REVIEW

**Review by kbergsund — APPROVED** [review]
Code written synchronously. Looks good. We can remove the dummy data now.

File: src/components/Context.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  //   } catch (err) {
+  //     console.log(err)
+  //   }
+  // }
+
+	// useEffect(() => {
+	// 	fetchPuzzles()
+	// 	// console.log('In effect>>>', puzzles)
+	// }, [])
+	useEffect(() => {
```
Comment (ccollums) [inline src/components/Context.tsx:54]: I am very curious how this difference changed it or if we can refactor to separate these out again

File: src/components/Puzzle.tsx [hunk 2]
```diff
@@ -3,21 +3,21 @@ import '../css/Puzzle.css';
...
+  missingPieces: string
+  price: string
   quality: string
-  size: number
+  size: string
   image: string
 }
 
-const Puzzle: React.FC<puzzleInfo> = ({ id, catagory, missingPieces, price, quality, size, image }) => {
+const Puzzle: React.FC<puzzleInfo> = ({ id, category, missingPieces, price, quality, size, image }) => {
```
Comment (ccollums) [inline src/components/Puzzle.tsx:15]: Ahhh my poor spelling causing an error I see haha

File: src/components/Puzzle.tsx [hunk 3]
```diff
@@ -3,21 +3,21 @@ import '../css/Puzzle.css';
 import PuzzleDetails from '../components/PuzzleDetails'
 
 interface puzzleInfo {
-  id: number
-  catagory: string
-  missingPieces: number
-  price: number
+  id: string
+  category: string
```
Comment (ccollums) [inline src/components/Puzzle.tsx:7]: So the errors happening were just because these were not correct??

**Review by ccollums — COMMENTED** [review]
Looks great! Glad we got this working

**Pull request merged**